.rubocop.yml

```diff
@@ -0,0 +1 @@
+inherit_from: .rubocop_todo.yml
```

Comment:
@GowthamK01 Please check if we need `rubocop_todo.yml`

Reply:
@gomathi-mallow Removed the rubocop_todo.yml as it is not a mandatory one and have configured the rubocop.yml for rubocop